src/blog/2024/11/building-uns-with-flowfuse.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Building a Unified Namespace (UNS) with FlowFuse
+subtitle: Implement your Unified Namespace seamlessly using our low-code platform
+description: Discover how FlowFuse helps you build a Unified Namespace (UNS) effortlessly, streamlining industrial data sharing, improving operational efficiency, and enabling real-time insights for smarter decision-making.
+date: 2024-11-28
+authors: ["sumit-shinde"]
+image: /blog/2024/11/images/building-uns-with-flowfuse.png
+keywords: building unified namespace using nodered, nodered uns, uns nodered
+tags:
+   - flowfuse
+   - uns
+   - mqtt
+---
+
+As systems and devices become more connected, managing data from different sources can be tricky. A [Unified Namespace (UNS)](/solutions/uns/) solves this by centralizing all your data in one place, making it easy to access and use.
+
+<!--more-->
+
+[FlowFuse](/) makes building a UNS simple. It connects old and new systems, collects data from devices and applications, and streamlines workflows. With tools like Node-RED for data flow, MQTT for real-time updates, and a central management layer, FlowFuse helps you improve efficiency and make better decisions.
+
+This article will show you how to build your UNS using FlowFuse, step by step.
+
+## **Building a UNS with Real-Time Sensor Data**
+
+This section explains how to set up a Unified Namespace (UNS) using FlowFuse, a Raspberry Pi, and an ADXL345 sensor. The Raspberry Pi collects data from the sensor, which we collect and process in Node-RED, calculate vibration magnitude, format it, and send it to the UNS using standardized topic names.
+
```

Comment:
If we're building an UNS, shouldn't we also do something with the data? Not just publish it?